src/workloads/docs/CollectionScanner.yml

```diff
@@ -70,6 +70,22 @@ Actors:
     Documents: 1000
     ScanType: standard
     GlobalRate: 10 per 10 seconds
+  - Duration: 2 minutes
+    # Similar to the previous phase but with an aggregation.
+    Documents: 1000
+    ScanType: standard
+    GlobalRate: 10 per 10 seconds
+    # Count the frequency of each character in array field "a".
+    AggregatePipeline:
+        {array: [{"$addFields":{"a":{"$map":{"input":{"$range":[0,{"$strLenCP":"$a"}]},"in":{"$substrCP":["$a","$$this",1]}}}}},
```

Comment:
"array" makes this tivial to implement (and would allow random stages to be generated).